src/components/sdkApi.tsx

```diff
@@ -0,0 +1,158 @@
+import {Fragment} from 'react';
+
+import {getCurrentPlatform} from 'sentry-docs/docTree';
+import {serverContext} from 'sentry-docs/serverContext';
+import {PlatformCategory} from 'sentry-docs/types';
+
+import {Expandable} from './expandable';
+import {codeToJsx} from './highlightCode';
+import {SdkDefinition} from './sdkDefinition';
+import {getPlatformHints} from './sdkOption';
+
+export interface ParameterDef {
+  name: string;
+  type: string | ObjectParameterDef;
+  defaultValue?: string;
+  description?: string;
+  required?: boolean;
+}
+
+type ObjectParameterDef = {
+  properties: ParameterDef[];
+  name?: string;
+};
+
+type Props = {
+  name: string;
+  parameters: ParameterDef[];
+  signature: string;
+  categorySupported?: PlatformCategory[];
+  children?: React.ReactNode;
+  language?: string;
+};
+
+export function SdkApi({
+  name,
+  children,
+  signature,
+  parameters = [],
+  language,
+  categorySupported = [],
+}: Props) {
+  const {rootNode, path} = serverContext();
+  const platform = getCurrentPlatform(rootNode, path);
+  const lang = language || platform?.language || 'typescript';
+
+  const {showBrowserOnly, showServerLikeOnly} = getPlatformHints(categorySupported);
+
+  return (
+    <SdkDefinition name={name} categorySupported={categorySupported}>
+      {showBrowserOnly && <div className="italic text-sm">Only available on Client</div>}
+      {showServerLikeOnly && (
+        <div className="italic text-sm">Only available on Server</div>
+      )}
+
+      <pre className="mt-2 mb-2 text-sm">{codeToJsx(signature, lang)}</pre>
+
+      {parameters.length ? (
+        <Expandable title="Parameters">
+          <div className="space-y-3">
+            {parameters.map(param => (
+              <ApiParameterDef key={param.name} language={lang} {...param} />
+            ))}
+          </div>
+        </Expandable>
+      ) : null}
+
+      {children}
```

Comment:
We need to play with the margins of the paragraphs of text that comes after the code/expandible. It needs to be closer I think. And after the text we need more whitespace.

Reply:
this is sadly pretty hard, because the text below "Parameters" is automatically converted to a <p>, which has `margin-top: 1.25em` - so no matter what margin-bottom we set for the expandable, there will always be at least this space there 😬 I do agree it would be nicer to have the spacing be a bit smaller there...

About the space after the text: Do you really feel that is not enough? Feels pretty "spacy" to me below the text 🤔 

![image](https://github.com/user-attachments/assets/0358a63f-1376-4204-85f5-1c335619d4d6)